.github/workflows/test.yml

```diff
@@ -76,7 +76,7 @@ jobs:
     steps:
       - uses: fkirc/skip-duplicate-actions@v5
         with:
-          cancel_others: 'true'
+          cancel_others: 'false'
```

Comment:
This comes from a different commit (https://github.com/prisma/prisma/commit/38318ed2544dc88d63f6d475354c24a2ef04caea). Is it intentional?